tools/phenomenal/isatools/mw2isa.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="isatab_validator" name="isatab_validator" version="1.0">
+  <description>Produce a validation report on an ISA-Tab ZIP archive</description>
```

Comment:
@pcm32 was in fact fixing this pb now, will push later in the day when I get the expected output